Dockerfile

```diff
@@ -49,3 +49,15 @@ ADD setup-user.sh /
 RUN chmod +x /docker-entrypoint.sh
 
 ENTRYPOINT /docker-entrypoint.sh
+
+# Compile and install PointCloud.
+# NOTE: release 1.2.0 would not build against PostgreSQL-11:
+# https://github.com/pgpointcloud/pointcloud/issues/248
+RUN apt-get -y update; apt-get -y install build-essential autoconf
+postgresql-server-dev-11 libxml2-dev zlib1g-dev
+RUN wget -O- \
+  https://github.com/pgpointcloud/pointcloud/archive/master.tar.gz \
+  | tar xz && \
+  cd pointcloud-master && \
+  ./autogen.sh && ./configure && make && make install && \
+  cd .. && rm -Rf pointcloud-master
```

Comment:
There a lot of other useful extensions that can be built for PostGIS, maybe we need to think adding a build argument so that a user can choose to build with or without. Maybe something like
`if  [ "$COMMUNITY_MODULES" == true ]; then`